frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/ReverificationResultHandler.java

```diff
@@ -89,6 +95,16 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                         AuditService.UNKNOWN,
                         PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()));
 
+        var idReverificationStateMaybe = idReverificationStateService.get(request.state());
+        if (idReverificationStateMaybe.isEmpty()) {
+            return generateApiGatewayProxyErrorResponse(400, ERROR_1061);
```

Comment:
Should we be doing some logging in both of these cases to differentiate whether the state doesn't exist or whether it doesn't match? (Possibly also some metrics, I don't know)

Reply:
Good shout, will have a look

Reply:
Adding error logs, but leaving metrics for now (we don't have any metrics for this lambda yet)